R/methods.R

```diff
@@ -1,17 +1,85 @@
-#' Title
+#' Imputation & analysis Methods
+#'
+#' @description
+#'
+#' These functions determine what methods rbmi should use when creating
+#' the imputation models, generating imputed values and  pooling the results.
 #'
 #' @name method
-#' @param burn_in TODO
-#' @param burn_between TODO
-#' @param same_cov TODO
-#' @param n_samples TODO
-#' @param covariance TODO
-#' @param threshold TODO
-#' @param REML TODO
-#' @param type TODO
-#' @param verbose TODO
-#' @param seed TODO
-#' @return TODO
+#'
+#' @param burn_in a numeric that specifies how many observations should be discarded
+#' prior to extracting actual samples. Note that the sampler
+#' is initialised at the maximum likelihood estimates and an uninformative
+#' prior is used thus in theory this value should not need to be that high.
+#'
+#' @param burn_between a numeric that specifies the "thinning" rate i.e. how many
+#' observations should be discarded between each sample. This is used to prevent
+#' issues associated with autocorrelation between samples.
+#'
+#' @param same_cov a logical, if true the MMRM model will be fitted using a single
```

Comment:
true -> `TRUE`
false -> `FALSE`

Reply:
PS: I have seen it also somewhere else. Also I have seen NULL instead of `NULL`.